sdks/python/apache_beam/pvalue.py

```diff
@@ -261,7 +266,7 @@ class TaggedOutput(object):
   """
 
   def __init__(self, tag, value):
-    if not isinstance(tag, string_types):
+    if not isinstance(tag, (bytes, unicode)):
```

Comment:
This is not equivalent to six.string_types.
I think this should be (str, unicode).
See https://pythonhosted.org/six/#six.string_types

Reply:
Fixed!